sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/MathExpressionsSuite.scala

```diff
@@ -403,11 +403,13 @@ class MathExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper {
 
   test("sqrt") {
     testUnary(Sqrt, math.sqrt, (0 to 20).map(_ * 0.1))
-    testUnary(Sqrt, math.sqrt, (-5 to -1).map(_ * 1.0), expectNaN = true)
+    testUnary(Sqrt, math.sqrt, (-5 to -1).map(_ * 1.0), expectNull = true)
```

Comment:
Yes, we migrated Hive and MySQL SQLs to Spark, and found some inconsistencies. `NaN` is unfamiliar to MySQL and Oracle users